src/plugins/plugin.decimation.js

```diff
@@ -152,6 +177,32 @@ function cleanDecimatedData(chart) {
   });
 }
 
+function getStartAndCountOfVisiblePointsSimplified(meta, points, animationsDisabled) {
+  const pointCount = points.length;
+
+  let start = 0;
+  let count;
+
+  const {iScale} = meta;
+  const axis = iScale.axis;
+  const {min, max, minDefined, maxDefined} = iScale.getUserBounds();
+
+  if (minDefined) {
+    start = _limitValue(Math.min(
+      _lookupByKey(points, iScale.axis, min).lo,
+      animationsDisabled ? pointCount : _lookupByKey(points, axis, iScale.getPixelForValue(min)).lo), 0, pointCount - 1);
```

Comment:
I think this can be simplified to just:
`start = _limitValue(_lookupByKey(points, iScale.axis, min).lo, 0, pointCount -1);`

The other part hast to do with pixels that we don't really know here yet (also its comparing pixel value to data value, which will not work as expected)

Same for max

Reply:
Wasn't sure about this, fixed